### PR TITLE
DEV-3817-exam-03-ref-cell-remake-exercise

### PR DESCRIPTION
**Pull request by mikysett**

### CODE REVIEW

File: subjects/car_rental/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+## car_rental
+
+### Instructions
+
+Create a struct named `Car` and another one named `RentalBusiness`.
+
+The scope of the exercise will be to modify and operate on the `Car` in the `RentalBusiness` even if this element is not declared as mutable.
+
```
Comment (nprimo) [inline subjects/car_rental/README.md:8]: I would add some description of the aspected behaviour of the `RentalBusiness` methods that need to be implemented

**Review by nprimo — CHANGES_REQUESTED** [review]
I feel the subject could be a bit more explicit about what it is asked to develop in this exercise

File: subjects/car_rental/README.md (hunk 1 above)
Comment (mikysett) [reply subjects/car_rental/README.md:10]: done

**Review by nprimo — APPROVED** [review]
LGTM!

**Pull request merged**